README.md

```diff
@@ -148,125 +113,126 @@ the `mbed_app.json` file.
+          "SPI_CLK": "D13",
+          "I2C_SDA": "D14",
+          "I2C_SCL": "D15",
+          "I2C_TEMP_ADDR":"0x90",
+          "I2C_EEPROM_ADDR":"0xA0",
+          "AIN_0": "A0",
+          "AIN_1": "A1",
+          "AIN_2": "A2",
+          "AIN_3": "A3",
+          "AIN_4": "A4",
+          "AIN_5": "A5",
+          "AOUT" : "A5",
+          "PWM_0": "D3",
+          "PWM_1": "D5",
+          "PWM_2": "D6",
+          "PWM_3": "D9",
+          "DEBUG_MSG": 0,
+          "DEVICE_SPI": 1,
+          "FSFAT_SDCARD_INSTALLED": 1
+     },
+     "target_overrides": {
+          "DISCO_F051R8": {
+               "SPI_MOSI": "SPI_MOSI",
+               "SPI_MISO": "SPI_MISO",
+               "SPI_CLK":  "SPI_SCK",
+               "SPI_CS":   "SPI_CS"
+          },
+          "K20D50M": {
+               "SPI_MOSI": "PTD2",
+               "SPI_MISO": "PTD3",
+               "SPI_CLK":  "PTD1",
+               "SPI_CS":   "PTC2"
+          },
+          "KL22F": {
+               "SPI_MOSI": "PTD6",
+               "SPI_MISO": "PTD7",
+               "SPI_CLK":  "PTD5",
+               "SPI_CS":   "PTD4"
+          },
+          "KL25Z": {
+               "SPI_MOSI": "PTD2",
+               "SPI_MISO": "PTD3",
+               "SPI_CLK":  "PTD1",
+               "SPI_CS":   "PTD0"
+          },
+          "KL43Z": {
+               "SPI_MOSI": "PTD6",
+               "SPI_MISO": "PTD7",
+               "SPI_CLK":  "PTD5",
+               "SPI_CS":   "PTD4"
+          },
+          "KL46Z": {
+               "SPI_MOSI": "PTD6",
+               "SPI_MISO": "PTD7",
+               "SPI_CLK":  "PTD5",
+               "SPI_CS":   "PTD4"
+          },
+          "K64F": {
+               "SPI_MOSI": "PTE3",
+               "SPI_MISO": "PTE1",
+               "SPI_CLK":  "PTE2",
+               "SPI_CS":   "PTE4"
+          },
+          "K66F": {
+               "SPI_MOSI": "PTE3",
+               "SPI_MISO": "PTE1",
+               "SPI_CLK":  "PTE2",
+               "SPI_CS":   "PTE4"
+          },
+          "LPC11U37H_401": {
+               "SPI_MOSI": "SDMOSI",
+               "SPI_MISO": "SDMISO",
+               "SPI_CLK":  "SDSCLK",
+               "SPI_CS":   "SDSSEL"
+          },
+          "LPC2368": {
+               "SPI_MOSI": "p11",
+               "SPI_MISO": "p12",
+               "SPI_CLK":  "p13",
+               "SPI_CS":   "p14"
+          },
+          "NUCLEO_L031K6": {
+               "SPI_MOSI": "SPI_MOSI",
+               "SPI_MISO": "SPI_MISO",
+               "SPI_CLK":  "SPI_SCK",
+               "SPI_CS":   "SPI_CS"
+          },
+          "nRF51822": {
+               "SPI_MOSI": "p12",
+               "SPI_MISO": "p13",
+               "SPI_CLK":  "p15",
+               "SPI_CS":   "p14"
+          },
+          "RZ_A1H": {
+               "SPI_MOSI": "P8_5",
+               "SPI_MISO": "P8_6",
+               "SPI_CLK":  "P8_3",
+               "SPI_CS":   "P8_4"
+          }
+     }
+}
+```
```

Comment:
IMO there's no need to duplicate the file, as we would have to update both README and the actual file.
Could replace the first mention of `mbed_app.json` above with a link, i.e. in markdown
```markdown
[`mbed_app.json`](./mbed_app.json)
```